components/automate-ui/src/app/pages/+compliance/+reporting/reporting-searchbar/reporting-searchbar.component.ts

```diff
@@ -432,7 +441,10 @@ export class ReportingSearchbarComponent implements OnInit {
   }
 
   requestForSuggestions(c: any): void {
```

Comment:
Please consider adding some strong typing to cut down on the use of `any` in this file. For this one, for example: define this type at the top of the file...
```
interface SuggestionEntry {
  text: string;
  type: string;
}
```
Then you can do this...
```suggestion
  requestForSuggestions(c: SuggestionEntry): void {
```
... so we know more clearly that `c.type` is a valid thing. (L444).

Reply:
Added type